src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ArrayConverter.cs

```diff
@@ -33,7 +32,81 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
 
             return base.ConvertTo(context, culture, value, destinationType);
         }
-    }
-}
 
+#if !NETSTANDARD10
+        /// <devdoc>
+        ///    <para>Gets a collection of properties for the type of array specified by the value parameter.</para>
+        /// </devdoc>
+        public override PropertyDescriptorCollection GetProperties(ITypeDescriptorContext context, object value, Attribute[] attributes)
+        {
+            PropertyDescriptor[] props = null;
+
+            if (value.GetType().IsArray)
```

Comment:
I added validation to return null in this case (documentation specifies a null return when no properties are found).